<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1274</width>
    <height>689</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1581</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CS 179 Project</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>1611</width>
      <height>931</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="A_Main_Menu">
     <widget class="QPushButton" name="Main_Menu_Load_Unload">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>80</y>
        <width>391</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Load/ Unload</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Main_Menu_Balance">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>280</y>
        <width>391</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Balance</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="D_Select_Load">
     <widget class="QPushButton" name="Select_Load_Confirm">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>350</y>
        <width>391</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>781</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Enter the containers you would like to Load:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="LoadContainerInput">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>731</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Enter Containers One at a Time Here</string>
      </property>
     </widget>
     <widget class="QListWidget" name="LoadContainerDisplay">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>731</width>
        <height>301</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="C_Select_Unload">
     <widget class="QPushButton" name="Select_Unload_Confirm">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>350</y>
        <width>391</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>911</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Select which containers you would like to unload:</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QListWidget" name="unLoadContainerDisplay">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>731</width>
        <height>301</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="waitlabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>821</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Please wait for containers to be displayed . . .</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="E_X_Moves_X_Minutes">
     <widget class="QPushButton" name="X_Moves_X_Minutes_Confirm">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>350</y>
        <width>391</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
     <widget class="QLabel" name="XMovesXminutestextdisplay">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>1231</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Calculating . . .</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="F_Step_X_of_X">
     <widget class="QPushButton" name="Step_X_of_X_Confirm">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>350</y>
        <width>391</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
     <widget class="QLabel" name="stepxofxdisplay">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>1231</width>
        <height>271</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Step X of X:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="weightinput">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>400</y>
        <width>621</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Enter Weight In Killograms Here</string>
      </property>
     </widget>
     <widget class="QLabel" name="weightprompt">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>340</y>
        <width>631</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Please enter the weight of the container below:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="G_Download_Manifest">
     <widget class="QPushButton" name="Download_Manifest_Confirm">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>350</y>
        <width>391</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
     <widget class="QLabel" name="downloadmanifestdisplay">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>1231</width>
        <height>311</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Please wait while manifest file is being saved . . .</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="B_Upload_Manifest">
     <widget class="QPushButton" name="Upload_Manifest_Confirm">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>350</y>
        <width>391</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>821</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Please upload the manifest for this operation</string>
      </property>
     </widget>
     <widget class="QPushButton" name="UploadManifestSelectFile">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>200</y>
        <width>271</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Select File</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>1271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>1271</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="Main_Window_Note">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Note</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Main_Window_Sign_In">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Sign In</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>1271</width>
      <height>50</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLineEdit" name="NoteInput">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="placeholderText">
        <string>Enter Note Here</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="UserNameInput">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="placeholderText">
        <string>Enter Name Here</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1271</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="ManifestDisplay">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Manifest Name Here:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="UserNameDisplay">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>User Name Here:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
